### PR TITLE
Add Cython utils needed for zero-copy import and export.

### PR DESCRIPTION
**Pull request by eriknw**
I'd like to add a few tests for this code, but it may be sufficient if `grblas` tests pass.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
     name='suitesparse-graphblas',
-    version='4.0.3',
+    version=versioneer.get_version(),
+    cmdclass=versioneer.get_cmdclass(),
     description='SuiteSparse:GraphBLAS Python bindings.',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     packages=find_packages(),
     author='Michel Pelletier, James Kitchen, Erik Welch',
+    author_email="jim22k@gmail.com,erik.n.welch@gmail.com",
```
Comment (michelp) [inline setup.py:56]: Go ahead and use michel@graphegon.com for my author email as we discussed.

**Review by michelp — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (eriknw) [reply setup.py]: Done.

**Comment by eriknw** [comment]
This allows `grblas` to be a pure Python package: https://github.com/metagraph-dev/grblas/pull/84

I still want to move some initialization logic for calling `GxB_init` to `suitesparse_graphblas` so that `pygraphblas` and `grblas` don't both try to initialize GraphBLAS.

**Comment by michelp** [comment]
This can also be done defensively, call GxB_Global_Option_get() and if it
GrB_PANICs then the library is not initialized.

**Comment by eriknw** [comment]
Whoops, I just saw your comment @michelp!  Yeah, each library could check using such a mechanism.  I defined `initialize` and `is_initialized` functions in `ssgb` (my current abbreviation for this package) instead.  Thanks for the trick!

`initialize` allows one to choose between using numpy's memory allocators or not.  I'm curious to do benchmarking to see if it makes a difference.

I think this is ready for final review and merging.

**Pull request merged**